File: WeLearnApp/src/main/java/com/welearn/WeLearnApp/dto/response/EvaluationResponse.java
package com.welearn.WeLearnApp.dto.response;

import lombok.*;
import lombok.experimental.FieldDefaults;

@Getter
@Setter
@Builder
@AllArgsConstructor
@NoArgsConstructor
@FieldDefaults(level = AccessLevel.PRIVATE)
public class EvaluationResponse {
    Integer star;
    String comment;
    String studentId;
    String studentName;
    String tutorId;
    String tutorName;
}

File: WeLearnApp/desktop-app/TutorApp/Models/ForAPI/Response/EvaluationResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TutorApp.Models.ForAPI.Response
{
    public class EvaluationResponse
    {
        public int star { get; set; }
        public string comment { get; set; }
        public string studentId { get; set; }
        public string studentName { get; set; }
        public string tutorId {  get; set; }
        public string tutorName { get; set; }
    }
}
